sonic/README.md

```diff
@@ -0,0 +1,29 @@
+# SONiC VM
+
+This is the vrnetlab docker image for SONiC's VM.
+The scripts in this directory are based on FreeBSD and VSRX kinds.
+
+> Available with [containerlab](https://containerlab.dev) as `vr-sonic` kind.
+
+## Building the docker image
+
+Download the latest `sonic-vs.img.gz` image from https://sonic.software/
+unompress and place it in this directory. Rename the file to `sonic-vs-[version].qcow2` and reference that file in the `Makefile`.
```

Comment:
since sonic.software is down, can you share with me the image you used for testing?

Reply:
It was working few days ago... anyway any generic qcow2 should work here as you already discovered :)